src/v2/Apps/Authentication/Server/redirectPostAuth.ts

```diff
@@ -0,0 +1,24 @@
+export function redirectPostAuth({ req, res }) {
+  const redirectTo = req.query["redirectTo"]
+
+  const configuredAllowedHosts =
+    process.env.ALLOWED_REDIRECT_HOSTS?.split(",") || []
```

Comment:
@damassi what is our preferred convention for accessing env vars that are not in sharify (i.e. server-side only)? I'm not sure if this is it.

Reply:
we should use `getENV`

Reply:
Though this also works!